<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>birdview</class>
 <widget class="QWidget" name="birdview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BirdView</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,20,1">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnOpenFile">
       <property name="text">
        <string>Open xls</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_file_name">
       <property name="text">
        <string>xls_name</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="vbox_bd"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hbox_btn_slider">
     <item>
      <layout class="QGridLayout" name="media_grid"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
